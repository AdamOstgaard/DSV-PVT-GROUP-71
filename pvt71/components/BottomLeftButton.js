
import React from "react";
import { StyleSheet, TouchableOpacity, Text } from "react-native";

export default class BottomLeftButton extends React.Component {
  render() {
    return (
      <TouchableOpacity style={styles.button} onPress={this.props.onPress}>
        <Text style={styles.textStyle}>{this.props.title}</Text>
      </TouchableOpacity>
    );
  }
}

const styles = StyleSheet.create({
  button: {
    backgroundColor: "#a9a9a9",
    width: "50%",
    height: 60
  },
  textStyle: {
    fontSize: 30,
    color: "#fff",
    textAlign: "left",
    paddingLeft: 10,
    paddingTop: 8
  }
});
