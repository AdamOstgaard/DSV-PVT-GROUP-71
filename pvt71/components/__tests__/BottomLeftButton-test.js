import "react-native";
import React from "react";
import BottomLeftButton from "../BottomLeftButton";
import renderer from "react-test-renderer";

describe("BottomLeftButton Test", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<BottomLeftButton title="Next" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
