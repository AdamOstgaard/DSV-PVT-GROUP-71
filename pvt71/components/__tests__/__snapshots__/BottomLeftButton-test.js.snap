// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomLeftButton Test renders correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#a9a9a9",
      "height": 60,
      "opacity": 1,
      "width": "50%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 30,
        "paddingLeft": 10,
        "paddingTop": 8,
        "textAlign": "left",
      }
    }
  >
    Next
  </Text>
</View>
`;
